autohooks/terminal.py

```diff
@@ -36,7 +36,7 @@
     "warning",
 )
 
-__term = None  # pylint: disable=invalid-name
+__term = Terminal()
```

Comment:
This Line was added to remove the possibility of an undefined _term (which was bothering mypy).
Now the following error occures:
`autohooks/terminal.py:113: error: "RichTerminal" has no attribute "print_overwrite"  [attr-defined]`
Which indeed seems to be true (at least i couldn't find this attribute). Any thoughts on this?

Reply:
I am really not sure about this change. When we apply this change the `_set_terminal` function should be rewritten too because it doesn't make sense to keep it as it is anymore. Personally I am in favor of not applying this change.